src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1742,6 +1742,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         ERR_ValueTupleResolutionAmbiguous3 = 37305
 
+        ERR_LineContWithCommentPreV161 = 37306
```

Comment:
>161 [](start = 35, length = 3)

nit: I assume `161` refers to a version. Let's avoid that.
Maybe `ERR_CommentAfterLineContinuationNotAvailable`

Reply:
Actually the 1 is "required" because this error has 1 parameter. the V16 is temporary name and has a PROTOTYPE where it is defined. I could change to ERR_CommentsAfterLineContinuationNotAvailable1. I am confused because the trailing number is not used consistently but it is in the comment about error names.

Reply:
Thanks for the clarification. 
Yes, the IDE has a convention to put the number of parameters in the name. We've only partially adopted that convention. `NotAvailable1` would be a suffix then.